inference-engine/tests/functional/inference_engine/ir_serialization/cleanup.cpp

```diff
@@ -49,15 +49,15 @@ TEST_F(SerializationCleanupTest, SerializationShouldWork) {
     ASSERT_TRUE(std::ifstream(m_out_bin_path, std::ios::in).good());
 }
 
-TEST_F(SerializationCleanupTest, SerializationShouldFail) {
+TEST_F(SerializationCleanupTest, SerializationShouldWorkWithDynamicFunction) {
```

Comment:
can we add explicit test that reads IR with PDS?

Reply:
This is exactly such a test! :) 
We create a function with an partially defined dimension in line 55.

Reply:
Oh. Sorry, I see what you mean.